mavros/src/plugins/sys_status.cpp

```diff
@@ -903,6 +912,19 @@ class SystemStatusPlugin : public plugin::PluginBase
 		}
 	}
 
+	/* -*- subscription callbacks -*- */
+
+	void statustext_cb(const mavros_msgs::StatusText::ConstPtr &req) {
+		mavlink::common::msg::STATUSTEXT statustext {};
+		ROS_WARN_COND_NAMED(req->text.length() >= statustext.text.size() - 1,
```

Comment:
Hmm, i'm a bit miss. Strings with 50 or more symbols will be truncated to 49.
So `s/>=/>/` or `s/\ -\ 1//`.

Reply:
oops, you're right